src/SnapKeyring.test.ts

```diff
@@ -59,6 +61,10 @@ describe('SnapKeyring', () => {
     },
   ] as const;
 
+  const keyringContext: KeyringExecutionContext = {
```

Comment:
```suggestion
  const executionContext: KeyringExecutionContext = {
```

Reply:
Yup, it's better, I've changed everywhere, thanks!